### PR TITLE
chore(snippets): add thirdparty-api-adapter snippets

### PR DESCRIPTION
**Pull request by kleyow**
part of mojaloop/mojaloop#372

### CODE REVIEW

File: thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: VerifyThirdPartyAuthorizationRequest
```
Comment (kleyow) [inline thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml:1]: Should this be renamed to ThirdPartyAuthorization**IDPost**Request to fall in line with other Mojaloop fspiop definitions?

File: thirdparty/openapi3/schemas/UpdateThirdpartyAuthorizationRequest.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+title: UpdateThirdpartyAuthorizationRequest
```
Comment (kleyow) [inline thirdparty/openapi3/schemas/UpdateThirdpartyAuthorizationRequest.yaml:1]: Should this be renamed to ThirdPartyAuthorization**IDPut**Request to fall in line with other Mojaloop fspiop definitions?

File: thirdparty/openapi3/schemas/AccountAddress.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+title: AccountAddress
```
Comment (kleyow) [inline thirdparty/openapi3/schemas/AccountAddress.yaml:1]: We have used `AccountId and `AccountAddress` interchangeably through out our services.

In reference to the original proposed specification change https://github.com/mojaloop/mojaloop-specification/issues/61 and https://github.com/mojaloop/account-lookup-service/commit/bdfa2117d8bb67aacb7740304dcd1eeb6e9d0ed1
which is my initial implementation of the proposed change in the ALS.

AccountAddress should be the correct name.

My question is do we align things like in example `sourceAccountId` in `ThirdpartyTransactionRequest` to be `sourceAccountId` here and throughout our services.

Discussion is open.

**Comment by kleyow** [comment]
thirdparty-api-adapter has 
- `VerifyThirdPartyAuthorizationRequest` (https://github.com/mojaloop/thirdparty-api-adapter/blob/master/src/interface/api-template.yaml#L323) 
- `UpdateThirdpartyAuthorizationRequest`(https://github.com/mojaloop/thirdparty-api-adapter/blob/master/src/interface/api-template.yaml#L356)

While auth-service has `ThirdPartyAuthorizationRequest` [https://github.com/mojaloop/auth-service/blob/master/src/interface/definitions.yaml#L231](url) as a super set of the two I believe. Not sure which direction we want to take.

Keep in mind my comments here
https://github.com/mojaloop/api-snippets/pull/16#discussion_r478547732
https://github.com/mojaloop/api-snippets/pull/16#discussion_r478547567

File: thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml (hunk 1 above)
Comment (lewisdaly) [reply thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml:1]: Yeah probably. My only thought is that `VerifyThirdPartyAuthorizationRequest` is a lot more descriptive

**Comment by lewisdaly** [comment]
> thirdparty-api-adapter has
> 
>     * `VerifyThirdPartyAuthorizationRequest` (https://github.com/mojaloop/thirdparty-api-adapter/blob/master/src/interface/api-template.yaml#L323)
> 
>     * `UpdateThirdpartyAuthorizationRequest`(https://github.com/mojaloop/thirdparty-api-adapter/blob/master/src/interface/api-template.yaml#L356)
> 
> 
> While auth-service has `ThirdPartyAuthorizationRequest` [https://github.com/mojaloop/auth-service/blob/master/src/interface/definitions.yaml#L231](url) as a super set of the two I believe. Not sure which direction we want to take.
> 
> Keep in mind my comments here
> [#16 (comment)](https://github.com/mojaloop/api-snippets/pull/16#discussion_r478547732)
> [#16 (comment)](https://github.com/mojaloop/api-snippets/pull/16#discussion_r478547567)

I think we should rename to `ThirdPartyAuthorizationIDPostRequest` and `ThirdPartyAuthorizationIDPutRequest`, but indeed keep them _separate_, since that allows us to express the validation a lot more strictly.

As for the auth service, it _only_ should receive a `ThirdPartyAuthorizationIDPostRequest`, and not have to worry about the `PUT` callback in it's swagger spec.

**Review by lewisdaly — APPROVED** [review]

File: thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml (hunk 1 above)
Comment (eoln) [reply thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml:1]: we need some convention

**Review by eoln — APPROVED** [review]
lgtm. maybe there is a need for some meetings to talk about naming convention show navigation over the code will be consistent?

**Comment by kleyow** [comment]
cool. made the changes. yes we can follow up with a naming convention meeting in the future.
this should be sufficient going forward until then.

**Pull request merged**